### PR TITLE
Article on broadcasting

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

File: content/notebooks/broadcasting.ipynb [hunk 1]
```diff
@@ -0,0 +1,626 @@
+{
```
Comment (pbrunet) [inline content/notebooks/broadcasting.ipynb:1]: Is it normal you commit this file too?

File: content/2016-05-25-broadcasting.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+# Benchmark
+
+The actual benchmark just runs each function through ``timeit`` for various array sizes.
+
+
+```python
+>>> import timeit
+>>> sizes = [1e3, 5e3, 1e4]
+>>> import pandas
```
Comment (pbrunet) [inline content/2016-05-25-broadcasting.md:179]: Ho, you start using pandas? :-)

File: content/2016-05-25-broadcasting.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+>>> import cython ; cython.__version__
+'0.24'
+```
+
+
+
+
+```python
+>>> import pythran; pythran.__version__
+'0.7.4.post1'
```
Comment (pbrunet) [inline content/2016-05-25-broadcasting.md]: Hum, not really, it is master, not the tagged one.

Comment (pbrunet) [inline content/2016-05-25-broadcasting.md]: What about the numba version?

**Comment by pbrunet** [comment]
Nice article !!!

**Comment by serge-sans-paille** [comment]
@pbrunet , @aguinet , @Pikalchemist : I've updated the article

File: content/2016-05-25-broadcasting.md [hunk 4]
```diff
@@ -282,103 +270,233 @@ numpy  1000 loops, best of 3: 1.79 ms per loop
...
+That's Pythran Leitmotiv: keep the Numpy abstraction, but try hard to make it run faster!
 
+# Round Two: Using the compiler
 
-That's Pythran Leitmotiv: keep the Numpy abstraction, but try hard to make it run faster!
+Gcc (and clang, and…) provide two flags that can be useful in this situation: ``-Ofast`` and ``-march=native``. The former is generally equivalent to ``-O3`` with a few extra flags, most noatbly ``-ffast-math`` that disregards standard compliancy with respect to floating point operation; In our case it makes it possible to reorder the operations to perform the final reduction using SIMD instructions. And with ``-march=native``, the code gets specialized for the host architecure. In the case of this blogpost, It means it can use [AVX](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions) and its 256bits vector register than can store four double precision floating!
```
Comment (pbrunet) [inline content/2016-05-25-broadcasting.md]: notably

File: content/2016-05-25-broadcasting.md [hunk 5]
```diff
@@ -282,103 +270,233 @@ numpy  1000 loops, best of 3: 1.79 ms per loop
...
+    <tr>
+      <th>10000.0</th>
+      <td>0.252430</td>
+      <td>0.090102</td>
+      <td>0.101917</td>
+      <td>0.097514</td>
+    </tr>
+  </tbody>
+</table>
+</div>
```
Comment (pbrunet) [inline content/2016-05-25-broadcasting.md:426]: It looks like not vectorized values.

Comment (serge-sans-paille) [inline content/2016-05-25-broadcasting.md:426]: oops s/scores/simd_scores/

**Pull request merged**